lib/models/stock_model.dart

```diff
@@ -66,15 +77,41 @@ String getDisplayTextForItemSize(ItemSize itemSize) {
       return "L";
     case ItemSize.medium:
       return "M";
-    case ItemSize.na:
-      return "x";
     case ItemSize.small:
       return "S";
     default:
       return "";
   }
 }
 
+// TODO: Use StockIdentifier
```

Comment:
Does this mean the `StockIdentifier` be used in the future?

Reply:
Currently the StockItem class looks like:

class StockItem {
  final ItemColor color;
  final ItemSize size;
  final int quantity;
}

In the future, I was thinking we could use StockIdentifier and change StockItem class to:

class StockItem {
  final StockIdentifier identifier;
  final int quantity;
}

The reason why I kept StockIdentifier is because I was using it for creating the has map. But, there are many issues with hash map in Dart. I ran into many issues while trying to refer to the object by hash values. So, I don't think there is any benefit of adding StockIdentifier to StockItem. I will remove this TODO comment.